app/controllers/views/index.erb

```diff
@@ -0,0 +1,38 @@
+<h1> Chitter </h1>
+
+<form action="/users/new">
+  <input type="submit" value="Sign up">
+</form>
+
+<% if @user %>
+
+  <h3> Welcome <%=@user.email%> </h3>
```

Comment:
wouldnt it make more sense to use @user.username here for real world practicality? For instance, Twitter would either have name or username. Great use of ruby interpolation however

Reply:
Yes that does seem more much logical! will definitely change it!